test/memory/test.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var leaky = require('leaky');
+var path = require('path');
+var assert = require('assert');
+var driver = leaky.start();
+var webdriver = require('selenium-webdriver');
+var results = [];
+
+function toggleMenu() {
+  driver.findElement(webdriver.By.css('#demo-menu-lower-left')).click();
+  driver.findElement(webdriver.By.css('#demo-menu-lower-left')).click();
+}
+
+console.log('Testing Menu Component');
+driver.get('file://' + path.join(__dirname, '../../dist/components/menu/demo.html'));
+toggleMenu();
+
+driver.sleep(1000);
```

Comment:
the leading and trailing sleeps are to let the initial and final animations finish :crossed_flags: